installation/resources/gardener/shoot.yaml.tpl

```diff
@@ -0,0 +1,74 @@
+kind: Shoot
+apiVersion: garden.sapcloud.io/v1beta1
+metadata:
+  name: __CLUSTER_NAME__
+  namespace: garden-__PROJECT_NAME__
+  annotations:
+    garden.sapcloud.io/purpose: __PURPOSE__
+  finalizers:
+    - gardener
+spec:
+  addons:
+    cluster-autoscaler:
+      enabled: true
+    heapster:
+      enabled: true
+    kubernetes-dashboard:
+      enabled: true
+    nginx-ingress:
+      enabled: true
+    monocular:
+      enabled: false
+  backup:
+    schedule: '*/5 * * * *'
+    maximum: 7
+  cloud:
+    profile: az
+    region: westeurope
+    secretBindingRef:
+      name: __NAME_OF_AZURE_SECRET__
+    seed: az-eu1
+    azure:
+      machineImage:
+        name: CoreOS
+        publisher: CoreOS
+        offer: CoreOS
+        sku: Stable
+        version: 1745.7.0
+      networks:
+        nodes: 10.250.0.0/19
+        pods: 100.96.0.0/11
+        services: 100.64.0.0/13
+        vnet:
+          cidr: 10.250.0.0/16
+        workers: 10.250.0.0/19
+      workers:
+        - name: kyma-worker
+          machineType: Standard_DS2_v2
+          autoScalerMin: 3
+          autoScalerMax: 4
+          volumeType: standard
+          volumeSize: 50Gi
+  dns:
+    provider: aws-route53
+    hostedZoneID: Z1ZC826MD6OQ78
```

Comment:
This can be different on different gardener environments. On staging I have such values:
```
hostedZoneID: Z17VJQYFX6YESD
domain: stage.cloudnext.shoot.staging.k8s-hana.ondemand.com
```
Mention it in the documentation that it works only on canary env. Optionally, check in the gardener how these values can be set/changed?

Reply:
It looks like hostedZoneID is assigned upon cluster deployment and does not need to be specified in shoot.yaml.